core/src/smartcontracts/isi/triggers.rs

```diff
@@ -61,6 +61,8 @@ pub mod isi {
                             .try_into()
                             .map_err(|e: &str| Self::Error::Conversion(e.to_owned()))?,
                     ),
+                    // `TriggerExecuted` event does not chain trigger executions at the moment
+                    FilterBox::TriggerExecuted(_) => unreachable!(),
```

Comment:
This is definitely reachable:
```rust
let trigger = Trigger::new(
    "test".parse()?,
    Action::new(
        vec![],
        Repeats::Indefinitely,
        "alice@wonderland".parse()?,
        TriggerExecutedEventFilter::new().into(),
    ),
);
client.submit_blocking(RegisterBox::new(trigger))?;
```

Maybe we should return an error instead indicating that chaining triggers isn't supported?

Reply:
Yes. That's exactly my concern mentioned in Possible Drawbacks

Reply:
Ah, sorry, I assumed "iroha developers" there, not "trigger developers".

I would say having a way for malicious client to induce panic in iroha node is a security issue and we shouldn't allow it

Reply:
Makes sense